app/javascript/packs/RequestPasswordPage.js

```diff
@@ -0,0 +1,21 @@
+import '@babel/polyfill'
+import 'core-js/es7/object'
+import { isBrowserIE11 } from 'utilities/ie11Utils'
+
+const isIE11 = isBrowserIE11(window)
+if (isIE11) {
```

Comment:
Maybe the entire conditional could be part of `utilities/ie11Utils` ?

Reply:
you could export directly a boolean `isIE11`

Reply:
Different PR better, out of scope... but I've seen it in at least 3 different componenets

Reply:
Yes, we can do it in it's own PR